meeting-minutes/client-minutes/march10-client-minutes.md

```diff
@@ -0,0 +1,53 @@
+
+1. Conduct "Stand Up".
+
+- Our Github project board will be displayed to show what we're planning on accomplishing during the upcoming sprint (unit testing, starting documentation, incorporating the client's suggestion). Our application will be run to demonstrate the modifications to the application in the past week (disabling the Mentor/Mentee in the admin tabs, routing the user to our application after Mentee Form submission).
+
+2. Discuss "Road Blocks"
+
+- Overall, it's been an extremely busy week for the team.
+
+3. Update the client on how we're linking organizations to the Mentor and Mentee Forms
+   - Currently, this done via a GET request by adding the admin's organization to the URL of the MentorForm in an email/the iFrame src property of the MenteeForm. This then gets extracted from the URL so that the user can be associated with the organization once the Mentee or Mentor Form has been successfully submitted. Matches are made based on Mentees and Mentors being from the same organization.
+     - The main concern is that Organization names are in the URL. Does the client see a way of allowing for the same functionality, without using a GET request? Keeping in mind that there's only 4 weeks left of the project, and so we might not be able to refactor extensively. However, we'll do our best to take client feedback into account.
+
+### NOTES
+
+Similar to previous weeks, the client called on our group first to update her on what we achieved over the last sprint. The client was walked through our application to demonstrate how the Mentee and Mentor forms are disabled in their respective tabs for a logged in admin. The email a mentor receives once an admin requests them was also shown to highlight how the URL now contains the name of the organization, as well as the updated URL for the iFrame tag.
+**In regard to Action Item 3 from above**:
+
+- Yes, passing information via a GET request is acceptable to associate invited Mentors and registered Mentees to the organization. The client didn't mind the organization's string being in the GET request, but she noted that we could possibly use the Organization's ID from our database instead (that way, multiple organizations can have the same name, but only be matched with the instance of the organization that the form was accessed through).
+  - Note: I realized after the meeting that our current setup in the database allows for organizations to be separated by their names (rather than how Mentees/Mentors/Admins have user IDs). I could email the client to see if leaving the organization name is acceptable, or if she prefers that we use a numeric ID instead. If she prefers a numeric ID, to save us from having to drastically change the setup of our database, we could possibly create an ID field in our database with a pseudo-random numeric for each organization. This could then replace the organization name. I'll give this some more thought to see if I can come up with any other alternatives.
```

Comment:
The website currently has many different places where we check organizations by name in the code. This is a big change. I think we shouldn't embark on it at this point, unless we want this to be one of the main focuses of the last weeks.

Reply:
Yea, I've been thinking about this since the client meeting and I definitely agree. From my conversation with the client, it sounded mainly like a suggestion. I'll let her know on Wednesday that this would be a significant thing to change. Judging by what's required for the Final deliverable, we should probably shift focus away from development and tackle requirements for project handover.